0ME9A/password
# Add History Page and Select All Button

**Title:** Add History Page and Select All Button

Hello,

I've implemented the new "/history" page and "Select All" button. Here are the details of the changes:

**Changes Made:**

1. Created a new "/history" page for password history. Users can access this page by clicking the "History" button from the "history" tab on the "/" page.
2. Optimized page load times by implementing on-demand loading for components.
3. Added a "Select All" button that allows users to delete all history items at once.

**How to Test:**

1. Navigate to the "/" page and click the "History" button in the "history" tab. Ensure that you're redirected to the "/history" page.
2. Check if the components on the "/history" page load on demand.
3. Generate some password history items, click the "Select All" button, and ensure that all history items are deleted.

I believe these features enhance the user experience by providing a dedicated page for password history and a convenient way to delete all history items. I've thoroughly tested these changes and everything seems to be working well. However, I'd appreciate if you could review my code and provide any feedback.

Thank you for considering this pull request. I'm looking forward to your feedback.

Best,
@0ME9A
